deploy/addons/ingress/ingress-configmap.yaml

```diff
@@ -19,7 +19,7 @@ data:
   hsts: "false"
 kind: ConfigMap
 metadata:
-  name: nginx-load-balancer-conf
+  name: nginx-configuration
```

Comment:
Why the name change? `nginx-load-balancer-conf` is pretty specific (maybe could be `nginx-ingress-configuration`) but `nginx-configuration` seems too generic.

Reply:
@balopat reverted the configmap back!